TKeazirian.HTTPServer/Handler/HealthCheckHandler.cs

```diff
@@ -0,0 +1,18 @@
+namespace TKeazirian.HTTPServer.Handler;
+
+using Response;
+using Request;
+
+public class HealthCheckHandler : Handler
+{
+    public override Response HandleResponse(Request request)
+    {
+        string body = File.ReadAllText("/Users/taylorkeazirian/Code/HTTPServer/http_server_spec/web/health-check.html");
```

Comment:
This is an absolute path; for code we push, it would be better to use a relative path, because then it should work for anyone who clones the repo, this includes CI CD and Server environments too.  Try to get this working with a relative path!